### PR TITLE
Add AMI

### PR DESCRIPTION
**Pull request by steveburton4**

### CODE REVIEW

File: hsm-client/hsm-client [hunk 1]
```diff
@@ -0,0 +1,104 @@
+#!/bin/sh
```
Comment (mattburgess) [inline hsm-client/hsm-client]: This, and the logrotate and sh file can just be deleted; the HSM client service is managed by the RPM that gets installed.

File: hsm-client/hsm-client_install.sh [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+# Adding in netcat and jq for troubleshooting
+sudo yum install -y nmap-ncat jq
+
+# Download & install AWS-CLI
+sudo pip install awscli
+
+# Install HSM client, the SDK and dependencies
+sudo yum install -y https://s3.amazonaws.com/cloudhsmv2-software/CloudHsmClient/EL7/cloudhsm-client-latest.el7.x86_64.rpm
+sudo yum install -y https://s3.amazonaws.com/cloudhsmv2-software/CloudHsmClient/EL7/cloudhsm-client-pkcs11-latest.el7.x86_64.rpm
```
Comment (mattburgess) [inline hsm-client/hsm-client_install.sh]: My reading of the docs suggests you don't need the pkcs11 or dyn RPMs.  Have I missed something?  We used to install the pkcs11 library back when we were trying to get things going with Python, but I think everything in Java land is handled by the jce RPM below.

File: hsm-client/hsm-client_install.sh [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+sudo yum install -y https://s3.amazonaws.com/cloudhsmv2-software/CloudHsmClient/EL7/cloudhsm-client-dyn-latest.el7.x86_64.rpm
+sudo yum install -y https://s3.amazonaws.com/cloudhsmv2-software/CloudHsmClient/EL7/cloudhsm-client-jce-latest.el7.x86_64.rpm
+
```
Comment (mattburgess) [inline hsm-client/hsm-client_install.sh]: Everything from here on down can be removed.  But can you put a `service cloudhsm-client disable` in here please? This way the userdata of the EC2 instance can 1) configure the client to point at the right cluster then 2) start the service.  Ta.

File: hsm-client/hsm-client (hunk 1 above)
Comment (benjaminsherwood) [reply hsm-client/hsm-client]: we want the logrotate, but the .sh can indeed just go, it's a placeholder we overwrite in userdata and userdata doesn't need it.

Comment (steveburton4) [reply hsm-client/hsm-client]: As discussed on slack, keeping logrotate as logs still produced

File: hsm-client/hsm-client_install.sh (hunk 3 above)
Comment (steveburton4) [reply hsm-client/hsm-client_install.sh]: Done

File: hsm-client/hsm-client_install.sh (hunk 2 above)
Comment (steveburton4) [reply hsm-client/hsm-client_install.sh]: Done

**Pull request merged**